azkaban-common/src/main/java/azkaban/executor/ExecutorLoader.java

```diff
@@ -35,6 +35,9 @@ ExecutableFlow fetchExecutableFlow(int execId)
   Map<Integer, Pair<ExecutionReference, ExecutableFlow>> fetchActiveFlows()
       throws ExecutorManagerException;
 
+  Pair<ExecutionReference, ExecutableFlow> fetchActiveFlowByExecId(int execId)
```

Comment:
No, we don't have it. When a flow is inactive, we just remove it from DB. We don't keep inactive flows info in DB.